src/Orleans.Runtime/Silo/Silo.cs

```diff
@@ -489,7 +483,7 @@ private void DoStart()
                     providerManagerSystemTarget.SchedulingContext)
                         .WaitWithThrow(initTimeout);
             runtimeClient.CurrentStreamProviderManager = siloStreamProviderManager;
-            if (logger.IsVerbose) { logger.Verbose("Stream provider manager created successfully."); }
+            if (logger.IsEnabled(LogLevel.Debug)) { logger.Debug("Stream provider manager created successfully."); }
```

Comment:
I see several of these, which were not introduced by you, but still: if not doing any string interpolation or something like that, then there is no need to check `IsEnabled`, it just adds a lot of noise

Reply:
Those are there also for perf reasons. If logging not set to debug level, then no need to create those logging strings at all.

Reply:
That's why I mentioned `if not doing any string interpolation or something like that` :) For constant strings, that optimization trick doesn't apply

Reply:
Ok. Anything else in this PR?